### PR TITLE
Improved OOPS Concept

### PR DESCRIPTION
**Pull request by tanuj1811**
Solved Issue ( ID : #196 )

### CODE REVIEW

File: java.md [hunk 1]
```diff
@@ -47,10 +47,6 @@ java example
 
 `//` - single line comment
 
-`/* */` - multi line comment
```
Comment (onecompiler-ops) [inline java.md:50]: Why these are removed?

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

**Review by tanuj1811 — COMMENTED** [review]
Approved Changes

**Comment by tanuj1811** [comment]
What should I do?
Make a new PR

**Comment by onecompiler-ops** [comment]
do rebase

**Comment by tanuj1811** [comment]
Hey @onecompiler-ops ,
Can you review again.

**Comment by tanuj1811** [comment]
Hey @onecompiler-ops ,
You haven't merged this PR yet. Is something wrong?

File: java.md [hunk 2]
```diff
@@ -349,18 +349,22 @@ class class_name {
...
-### 3. Abstraction
+Oject Oriented Programming (OOPS) basically stands on 4 main principles. All of them are mentioned in detail below:  
 
-Data abstraction is a technique which provides only the required data to be visible or accessible to outside world. 
-`abstract` keyword is used for classes and methods. 
+### 1. Abstraction
+
+Abstraction is a technique provides that only the required infomation to be visible to outside world, all other are hidden. 
 
+Example : Consider the Paytm app, we don't know how money is being transferred from one account to another within a second, all we know is we can make payment using the app.
```
Comment (onecompiler-ops) [inline java.md:365]: Please use global examples

**Comment by onecompiler-ops** [comment]
This is a very specific use-case, and does not belong in Cheat Sheets, please write this as a post instead here https://onecompiler.com/posts

**Comment by tanuj1811** [comment]
I just improved the quality of sheet by adding points or examples. 
Post is like blog. Both are different. You should read again. These changes more understandable and knowledable.

Thank you

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

**Comment by onecompiler-ops** [comment]
@tanuj1811 cheat sheets are supposed to be precise, please avoid lengthy explanations. Keep them simple and easy with meaningful code snippets. 
In Tutorials we can add in-depth details

**Comment by tanuj1811** [comment]
@onecompiler-ops , 
I updated the example. Now the example is more concise and simple. You can review it.

Thanks

**Comment by onecompiler-ops** [comment]
@tanuj1811 can you have a look at the Tutorials repo? Your content better suites there

**Pull request closed without merging**